tests/test_notify_buyers_when_requirements_close.py

```diff
@@ -37,7 +37,7 @@ def check_date_closed(value, expected):
         (None, datetime.date(2015, 1, 1)),
         ('2016-01-02', datetime.date(2016, 1, 2))
     ]:
-        yield check_date_closed, value, expected
+        check_date_closed(value, expected)
```

Comment:
I guess this could have been replaced with a parameterized test, but I'm honestly not sure there's much benefit. https://docs.pytest.org/en/latest/parametrize.html